src/TuDa.CIMS.Web/Components/FilterComponent.razor

```diff
@@ -0,0 +1,77 @@
+﻿@* FilterComponent *@
```

Comment:
Sorry for the inconvenience, but you need to change the name of the component and file to `Filter` like I described in #21